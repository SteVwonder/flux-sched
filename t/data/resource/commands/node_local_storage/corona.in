match allocate @TEST_SRCDIR@/data/resource/jobspecs/node_local_storage/corona.yaml
match allocate @TEST_SRCDIR@/data/resource/jobspecs/node_local_storage/corona.yaml
quit
