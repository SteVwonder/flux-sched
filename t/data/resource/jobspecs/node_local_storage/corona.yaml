version: 1
resources:
  - type: node
    count: 1
    with:
      - type: slot
        label: default
        count: 1
        with:
          - type: socket
            count: 1
            with:
              - type: core
                count: 1
              - type: storage
                count: 1500

# a comment
attributes:
  system:
    duration: 57600
tasks:
  - command: [ "default" ]
    slot: default
    count:
      per_slot: 1
