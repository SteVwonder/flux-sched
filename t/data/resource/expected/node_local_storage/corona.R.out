      ------------core0[1:x]
      ------------storage0[1562:x]
      ---------socket0[1:x]
      ------corona116[1:s]
      ---cluster0[1:s]
INFO: =============================
INFO: JOBID=1
INFO: RESOURCES=ALLOCATED
INFO: SCHEDULED AT=Now
INFO: =============================
INFO: =============================
INFO: No matching resources found
INFO: JOBID=2
INFO: =============================
